pkg/certificate/controller.go

```diff
@@ -43,19 +43,43 @@ func (m *Manager) add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return object.GetName() == m.webhookName
 	}
 
+	isGeneratedSecret := func(object client.Object) bool {
+		if object.GetName() == m.caSecretKey().Name {
+			return true
+		}
```

Comment:
can we move [this](https://github.com/qinqon/kube-admission-webhook/pull/60/files#diff-477009fdb255fbbfc90de2def9ee37495b402bf4e99058b8fdf409341c11f066R47-R49) to a specific function `isCASecretRelated` (or any better name you can find)